presto-client/src/main/java/com/facebook/presto/client/ClientTypeSignatureParameter.java

```diff
@@ -144,7 +144,7 @@ public int hashCode()
         private static final ObjectMapper MAPPER = new JsonObjectMapperProvider().get();
 
         @Override
-        public ClientTypeSignatureParameter deserialize(JsonParser jp, DeserializationContext ctxt)
+        public ClientTypeSignatureParameter deserialize(JsonParser jp, DeserializationContext ctx)
```

Comment:
This seems to be the preferred spelling (in this repository, in general, and personally).